.github/workflows/publish.yml

```diff
@@ -0,0 +1,44 @@
+name: Publish to NPM
+
+on:
+  release:
+    types: [published]
```

Comment:
Should this be on tagged versions instead? I'm unfamiliar with `types: [published]`. Also, should we use release-it for this repo instead of auto publishing?